packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -661,10 +774,11 @@ contract Deploy is Deployer {
                     abi.encodeCall(
                         SuperchainConfig.initialize,
                         (
-                            cfg.finalSystemOwner(), // systemOwner
-                            cfg.superchainConfigInitiator(), // initiator
+                            // todo: fix this
```

Comment:
Wondering about this todo

Reply:
My bad. 
The thing that needed to be fixed was the fact that if the initiator address is set in the config, then it won't be in the state when this script is run on a devnet. 

The fix is the code in `setupSuperchain` that deploys a new Safe if one is not provided.

I'll remove this todo.